scripts/dot-createlicense.ps1

```diff
@@ -14,6 +14,14 @@ function CreateLicence {
     )
 
     $Cloud = (Get-ItemProperty -Path HKLM:\Software\LANSA  -Name 'Cloud').Cloud
+    $LicenseName = Split-Path $licenseFile_ -leaf
+    if (($Cloud -eq 'AWS') -and ($LicenseName -in @('LANSAScalableLicense.pfx','LANSAIntegratorLicense.pfx'))) {
```

Comment:
No need to condition on the filename in here. That way it will work with whatever the caller asks to load.

Reply:
If $IntallIDE -eq true, then "Create-Licence LansaDevelopmentLicense" will be called. But you said LansaDevelopmentLicense should not be retrieved from parameter store. That's why I conditioned it.

Reply:
It won't be because $InstallIDE is false!
If we ever resurrect $InstallIDE = $true we can fix it then